pkgs/misc/emulators/wine/base.nix

```diff
@@ -6,7 +6,8 @@
   buildScript ? null, configureFlags ? ""
 }:
 
-assert stdenv.cc.cc.isGNU or false;
+assert stdenv.isLinux || stdenv.isDarwin;
```

Comment:
I think we can just remove this. The assert is only needed to prevent a potential evaluation failure below.

Reply:
yeah do `meta.platforms` instead.

Reply:
platform.unix + platform.windows, or all or something.